pay-api/src/pay_api/services/eft_short_names.py

```diff
@@ -98,17 +99,33 @@ def create_shortname_link(cls, short_name_id: int, auth_account_id: str, **kwarg
     def get_shortname_links(cls, short_name_id: int) -> List[EFTShortnameLinksModel]:
         """Get EFT short name account links."""
         current_app.logger.debug('<get_shortname_links')
-        query = db.session.query(EFTShortnameLinksModel) \
-            .filter(EFTShortnameLinksModel.eft_short_name_id == short_name_id) \
-            .filter(EFTShortnameLinksModel.status_code.in_([EFTShortnameStatus.LINKED.value,
-                                                            EFTShortnameStatus.PENDING.value])) \
-            .order_by(EFTShortnameLinksModel.created_on.asc())
+        statement_summary_query = cls.get_statement_summary_query().subquery()
+        query = db.session.query(EFTShortnameLinksModel.id.label('id'),
```

Comment:
any reason for the specific column? can't you just return the entire object considering it's not going to be a whole lot of rows?

Reply:
Will double check this, I think I was running into a tuple issue on this particular query. Might not be an issue after I cleaned it all up.

Reply:
all good very minor

Reply:
Tried switching back to the model and I got an oddly structured result.

Will do a bit of digging, why it behaves this way. Leaving this as is for this PR.